evms/evmone.go

```diff
@@ -137,6 +137,11 @@ func (evm *EvmoneVM) Copy(out io.Writer, input io.Reader) {
 				continue
 			}
 		}
+		// Evmone sometimes report a negative refund
+		if i := bytes.Index(data, []byte(`"refund":-`)); i > 0 {
```

Comment:
Anyway, wouldn't it be better to change the parting to signed `int64`?

Reply:
The type is imported from `github.com/ethereum/go-ethereum/eth/tracers/logger/logger.go`, it's not defined here. I preferred to keep it instead of copy-pasting the definition.